web_group_expand/i18n/ca.po

```diff
@@ -0,0 +1,31 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* web_group_expand
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```

Comment:
`Odoo Server 14.0\n`? Same for the other PO files.

Reply:
done, thanks